plugins/color_panel/macos/FLEColorPanelPlugin.mm

```diff
@@ -33,7 +33,7 @@ - (NSString *)channel {
 - (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
   BOOL handled = YES;
   if ([call.methodName isEqualToString:@(plugins_color_panel::kShowColorPanelMethod)]) {
-    [self showColorPanel];
+    [self showColorPanel: call.arguments];
```

Comment:
No spaces after : in ObjC. (clang-format should take care of things like this for you.)